docs/GettingStarted.rst

```diff
@@ -1324,7 +1324,7 @@ You can display the pinned articles with one of the following layouts :
 
 |image198|
 
-When you click on the "See all" button, you are redirected automatically to news application filtered by pinned articles.
+When you click on the "See all" button,  you are redirected automatically to news application filtered by pinned articles
```

Comment:
The changes on this sentence should be reverted IMO.

Reply:
fixed.